src/wp-includes/functions.php

```diff
@@ -7635,3 +7633,85 @@ function is_php_version_compatible( $required ) {
 function wp_fuzzy_number_match( $expected, $actual, $precision = 1 ) {
 	return abs( (float) $expected - (float) $actual ) <= $precision;
 }
+
+/**
+ * Formats attributes string from associative array $attributes to be placed insed `<script>` tag.
+ *
+ * Automatically injects type attribute if needed.
+ * Used by wp_print_script_loader_tag() and wp_print_inline_script_tag().
+ *
+ * @param array     $attributes     Optional. `<script>` tag attributes.
+ * @param bool      $double_quotes  Optional. Uses double quotes `"` to wrap attribute values if true, single quotes `'` otherwise.
+ * @return string  Inline JavaScript code wrapped around <script> tags if $echo is true, null otherwise.
+ */
+function wp_sanitize_script_attributes( $attributes = array() ) {
+	if ( ! array_key_exists( 'type', $attributes ) && ! is_admin() && ! current_theme_supports( 'html5', 'script' ) ) {
+		$attributes['type'] = 'text/javascript';
+	}
+	/**
+	 * Filters script attributes.
+	 *
```

Comment:
```suggestion
	 *
	 * @since 5.6.0
	 *
```

All functions and filters would need a `@since` annotation